src/main/java/com/msrvtasks/springboottasks/SpringBootTasksApplication.java

```diff
@@ -2,7 +2,9 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Configuration;
 
+@Configuration
```

Comment:
rather have @Configuration on your configuration java class and not your application class in order to separate your code out to follow the single responsibility SOLID principal.